crates/bevy_core_pipeline/src/lib.rs

```diff
@@ -35,11 +37,29 @@ use bevy_render::{
 
 /// Resource that configures the clear color
 #[derive(Clone, Debug)]
-pub struct ClearColor(pub Color);
+pub struct ClearColor {
+    pub default_color: Color,
+    pub per_target: HashMap<RenderTarget, Color>,
```

Comment:
I would like some docs on this field: its use is not immediately obvious.

Reply:
Done.